src/renderer/features/notifications/NotificationsList/ui/notifies/MultisigCreatedNotification.tsx

```diff
@@ -5,13 +5,14 @@ import type { MultisigCreated } from '@shared/core';
 import { WalletType } from '@shared/core';
 import { BodyText } from '@shared/ui';
 import { useI18n } from '@app/providers';
+import { ChainTitle } from '@entities/chain';
 
 type Props = {
   notification: MultisigCreated;
 };
 
 export const MultisigCreatedNotification = ({
-  notification: { threshold, signatories, multisigAccountName },
+  notification: { threshold, signatories, multisigAccountName, chainId },
```

Comment:
Not sure about this change for instance, it doesn't seem to have anything to do with matrix